tests/Microsoft.Bot.Streaming.Tests/StreamingRequestHandlerTests.cs

```diff
@@ -348,5 +342,29 @@ public override Task SendAsync(ArraySegment<byte> buffer, WebSocketMessageType m
                 throw new NotImplementedException();
             }
         }
+
+        private class MessageBot : IBot
+        {
+            public async Task OnTurnAsync(ITurnContext turnContext, CancellationToken cancellationToken = default) => await turnContext.SendActivityAsync(MessageFactory.Text("do.not.go.gentle.into.that.good.night"));
+        }
+
+        private class FakeContentStream : IContentStream
+        {
+            public FakeContentStream(Guid id, string contentType, Stream stream)
+            {
+                Id = id;
+                ContentType = contentType;
+                Stream = stream;
+                Length = int.Parse(stream.Length.ToString());
+            }
+
+            public Guid Id { get; set; }
+
+            public string ContentType { get; set; }
+
+            public int? Length { get; set; }
+
+            public Stream Stream { get; set; }
+        }
```

Comment:
This was moved after StreamingRequestHandlerTests.FauxSock was made public so that the tests would build.